src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
+    </h2>
```

Comment:
maybe it will be better if you will use single line? what do you think?
```suggestion
<h2 className="Person__name">
  {`My name is ${person.name}`}
</h2>
```